packages/experiments/src/components/Button/Button.view.tsx

```diff
@@ -22,6 +22,7 @@ export const ButtonView: IButtonComponent['view'] = props => {
   const _onClick = (ev: React.MouseEvent<HTMLElement>) => {
     if (!disabled && onClick) {
       onClick(ev);
+      ev.stopPropagation();
```

Comment:
Is this consistent with existing buttons in OUFR? Seems a bit different.

Also linse 27-29 are weird. They're basically no-op.

Reply:
I think lines 27-29 were something that was left after I split `MenuButton` into its own variant. I'll remove them. Also, I think you're right, in the current button `ev.stopPropagation()` only occurs `onMenuClick`, so I'll also remove that from here.